### ISSUE TITLE
Bug: `no-invalid-at-rules` complains about Tailwind at-rules

### ISSUE THREAD

**Issue by jrmajor** [eslint/css#219]
### Environment

ESLint version: 9.32.0
@eslint/css version: 0.10.0
Node version: 20.19.1
npm version: 10.8.2
Operating System: linux 5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```js
import css from '@eslint/css';
import { defineConfig } from 'eslint/config';
import { tailwind4 } from 'tailwind-csstree';

export default defineConfig([
	{
		files: ['**/*.css'],
		language: 'css/css',
		plugins: { css },
		languageOptions: {
			syntax: tailwind4,
		},
		extends: [css.configs.recommended],
	},
]);

```
</details>

```css
@import 'tailwindcss';

@plugin 'tailwindcss-animate';

@source not inline('container');

@theme {
  --font-sans: InterVariable, ui-sans-serif, system-ui, sans-serif;
}

```

### What did you expect to happen?

No `css/no-invalid-at-rules` errors, because I've set syntax to Tailwind 4.

### What actually happened?

```
/home/projects/vitejs-vite-ppttfe8f/src/style.css
  3:1  error  Unknown at-rule '@plugin' found  css/no-invalid-at-rules
  5:1  error  Unknown at-rule '@source' found  css/no-invalid-at-rules
  7:1  error  Unknown at-rule '@theme' found   css/no-invalid-at-rules
```

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/eslint-tailwind-4-at-rules-repro?file=src%2Fstyle.css

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

I have no idea what you mean by „Which language are you using?” question in the issue template. I guess I use CSS, but there is no such option in the dropdown. There is also nothing about „language mode” in readme, except for „tolerant mode”, which also isn't an option.
